emails/sns.py

```diff
@@ -55,13 +54,11 @@
 def verify_from_sns(json_body):
     pemfile = _grab_keyfile(json_body['SigningCertURL'])
     cert = crypto.load_certificate(crypto.FILETYPE_PEM, pemfile)
-    signature = base64.decodestring(six.b(json_body['Signature']))
+    signature = base64.decodebytes(json_body['Signature'].encode('utf-8'))
 
     hash_format = _get_hash_format(json_body['Type'])
 
-    crypto.verify(
-        cert, signature, six.b(hash_format.format(**json_body)), 'sha1'
-    )
+    crypto.verify(cert, signature, hash_format.format(**json_body), 'sha1')
```

Comment:
```suggestion
    crypto.verify(cert, signature, hash_format.format(**json_body).encode('utf-8), 'sha1')
```
`crypto.verify()` does [handle](https://github.com/pyca/pyopenssl/blob/master/src/OpenSSL/crypto.py#L2849) the text and convert to a byte object but it returns a warning. Might as well just pass a byte object?

Reply:
Done. But I re-formatted the line to make it fit.